pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.isort]
+profile = "black"
+sections = [
+  "FUTURE",
+  "STDLIB",
+  "THIRDPARTY",
+  "ARCHSPEC", "LLNL", "FIRSTPARTY",
+  "LOCALFOLDER",
+]
+known_first_party = "spack"
+known_archspec = "archspec"
+known_llnl = "llnl"
+src_paths = "lib"
+force_sort_within_sections = false
```

Comment:
This is the default, so we could remove it.

Reply:
done